src/screens/track-screen/TrackScreenStats.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback } from 'react'
+
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import { StyleSheet, View } from 'react-native'
+
+import IconFavorite from 'app/assets/images/iconHeart.svg'
+import IconRepost from 'app/assets/images/iconRepost.svg'
+import Text from 'app/components/text'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { flexRowCentered } from 'app/styles'
+import { ThemeColors } from 'app/utils/theme'
+
+import { TrackScreenStat } from './TrackScreenStat'
+
+const messages = {
+  plays: 'Plays'
+}
+
+type TrackScreenStatsProps = {
+  favoriteCount: number
+  playCount?: number
+  repostCount: number
+  showFavoriteCount: boolean
+  showListenCount: boolean
+  showRepostCount: boolean
+  trackId: ID
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    statsContainer: {
+      ...flexRowCentered(),
+      justifyContent: 'center',
+      marginBottom: 12
+    },
+    countLabel: {
+      fontSize: 16,
+      color: themeColors.neutralLight4,
+      textAlign: 'center'
+    }
+  })
+
+/**
+ * The stats displayed on track and playlist screens
+ */
+export const TrackScreenStats = ({
+  favoriteCount,
+  playCount = 0,
+  repostCount,
+  showFavoriteCount,
+  showListenCount,
+  showRepostCount,
+  trackId
+}: TrackScreenStatsProps) => {
+  const styles = useThemedStyles(createStyles)
+  const onPressFavorites = useCallback(() => {
+    // TODO: navigate to favorites
+    // goToFavoritesPage(trackId)
+  }, [trackId])
+
+  const onPressReposts = useCallback(() => {
+    // TODO: navigate to reposts
+    // goToRepostsPage(trackId)
+  }, [trackId])
+
+  return (
+    <>
+      {(showListenCount || showFavoriteCount || showRepostCount) && (
```

Comment:
just a small callout, been chatting with some other folks about `&& vs ? < /> : null` curious what your thoughts are. here is the blog post that got me thinking about it: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
we did talk about this before fwiw https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#f1c89a4e0843430bbf98f0ce53dbcf14